cli/azd/pkg/tools/github/github.go

```diff
@@ -151,6 +151,10 @@ func (cli *ghCli) Name() string {
 	return "GitHub CLI"
 }
 
+func (cli *ghCli) GetBinaryPath() string {
```

Comment:
NIT: In general, go APIs eschew the `Get` Prefix.  I think a better name would just be `BinaryPath`.  We're somewhat inconsistent here (almost certainly due to yours truly and his fingers still being stuck in .NET land when we started this project) but I've been trying to not introduce new uses of `Get` (and clean up the old uses when refactoring).

No need to turn the crank on this if there are no other changes, but if you end up pushing more commits, might be nice to fix.